version.go

```diff
@@ -21,11 +21,19 @@ func (v versionChecker) Checks() []fthealth.Check {
 	versionCh = make(chan result)
 	go loop(latest, 300, versionCh)
 	check := fthealth.Check{
-		BusinessImpact:   "A part of the publishing workflow might be affected",
+		BusinessImpact:   "It may be possible to compromise our publishing stack using a known security vulnerability.",
 		Name:             "CoreOS version",
-		PanicGuide:       "Please refer to technical summary",
-		Severity:         2,
-		TechnicalSummary: "CoreOS version is out of date or cannot be determined",
+		PanicGuide:       `* Visit https://coreos.com/releases/ and look at the release notes for the latest version.
+* Check whether any Security Fixes have been included.
+* For each security fix, check its CVSS score.
+* If any security fix has a CVSS score higher than 7, then this is a Critical Update.
+* Any updates which aren't critical can be acknowledged out-of-hours and an email sent to the Universal Publishing team.
+* Critical Updates should be escalated to Technical Operations.
```

Comment:
I like the proposal. Can you please include link to wiki too? https://github.com/Financial-Times/coreos-upgrade/wiki